### PR TITLE
Add delete and redeploy buttons to the admin page.

### PR DESCRIPTION
**Pull request by davidread**
screen shot 2017-10-10 at 11 59 27

### CODE REVIEW

File: sandboxmgt/app/templates/request_by_admin.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 form - form object
 {% endcomment %}
 
-<form action="/sandboxes" method="post">
+<form action="/admin" method="post">
```
Comment (davidread) [inline sandboxmgt/app/templates/request_by_admin.html:7]: This is an unrelated fix for this form, that I should have done previously.

**Review by andreagrandi — APPROVED** [review]
